ws/src/server_builder.rs

```diff
@@ -54,10 +57,11 @@ impl ::std::error::Error for Error {
     }
 
     fn cause(&self) -> Option<&::std::error::Error> {
-        Some(match *self {
-            Error::Io(ref e) => e,
-            Error::WebSocket(ref e) => e,
-        })
+        match *self {
+            Error::Io(ref e) => Some(e),
+            Error::WebSocket(ref e) => Some(e),
+			Error::ConnectionClosed => None,
```

Comment:
yea, it uses tabs like the rest of the codebase while the file (mistakenly) uses spaces